event_handler.go

```diff
@@ -94,6 +94,7 @@ func eventHandler(ctx *IrcContext, rtm *slack.RTM) {
 			text := ev.Msg.Text
 			for _, attachment := range ev.Msg.Attachments {
 				text = joinText(text, attachment.Pretext, "\n")
+				text = joinText(text, attachment.Title, "\n")
```

Comment:
Shouldn't this only be done if `attachment.Title != ""`?

Reply:
`joinText` function will take care of appending it to `text` only if it exists